tier4_localization_msgs/srv/InitializeLocalization.srv

```diff
@@ -0,0 +1,20 @@
+# constants for specifying initialization method
+uint8 AUTO = 0
+uint8 DIRECT = 1
+
+# input pose for initialization
+geometry_msgs/PoseWithCovarianceStamped[<=1] pose_with_covariance
+
+# Variable to set initialization method
+# AUTO: The initial position is automatically estimated with localization algorithm.
+#              The input pose will be used as an initial guess if provided.
```

Comment:
Sorry, my suggestion seemed to include weird indent. 
Could you remove unnecessary space?

Reply:
The content looks okay so I'm approving the PR.

Reply:
OK! I have removed them https://github.com/tier4/tier4_autoware_msgs/pull/121/commits/f9f806764d5f023254e63c3dd3fb56fb89aa5a0a